src/Core/src/PropertyMapper.cs

```diff
@@ -169,12 +222,22 @@ public void Add(string key, Action<TViewHandler, TVirtualView> action) =>
 			SetPropertyCore(key, (h, v) =>
 			{
 				if (v is TVirtualView vv)
+				{
 					action?.Invoke((TViewHandler)h, vv);
+				}
 				else if (Chained != null)
 				{
 					foreach (var chain in Chained)
 					{
-						if (chain.GetProperty(key) != null)
+						// Try to leverage our internal method which uses merged mappers
+						if (chain is PropertyMapper propertyMapper)
+						{
+							if (propertyMapper.TryUpdatePropertyCore(key, h, v))
```

Comment:
This is a split if block, so if it is a property mapper but the update fails it will go to the next chained mapper? Should it be merged into a single one so that it can fall back to the Get/Update block? Or is it intentional that if it is a PropertyMapper then we try and skip?

Reply:
It is intentional, because a `ƒalse` means that the mapper doesn't handle that key at all (no mappers registered for that property), so it'd be useless to invoke `GetProperty(key)` which recursively checks all of the `chain`'s chained mappers.